style.md

```diff
@@ -965,6 +966,82 @@ func (f *foo) isRunning() bool {
 </td></tr>
 </tbody></table>
 
+### Avoid Mutable Globals
+
+Avoid mutating global variables, instead opting for dependency injection.
+This applies to function pointers as well as other kinds of values.
```

Comment:
nit: We might want to expand this to include "don't introduce mutable global variables" in addition to "don't mutate global variables". Without getting overly specific, this could look something like:

> Avoid mutating global variables, instead opting for dependency injection.
> This applies to any assignable values that could be used as global state,
> such as function pointers, strings, and other kinds of values. Errors are
> a notable exception to this guidance.
>
> Similarly, avoid introducing mutable global variables by using `const` where possible.

The last part, in particular, is probably worth its own quick section - i.e., `const` wherever possible.

Reply:
(also, should've clarified - I made some minor tweaks to reference "assignable values" given that function pointers are perhaps not representative enough of what could be considered "mutable global state", given how folks often use strings or scalars to serve as "static config" in addition to overriding global function vars.)